dask/tests/test_distributed.py

```diff
@@ -343,6 +359,10 @@ def test_blockwise_dataframe_io(c, tmpdir, io, fuse, from_futures):
     pd = pytest.importorskip("pandas")
     dd = pytest.importorskip("dask.dataframe")
 
+    # TODO: this configuration is flaky on osx in CI
+    if from_futures is True and fuse is False and sys.platform == "darwin":
```

Comment:
We've also seen this test fail when `fuse=None` (xref https://github.com/dask/dask/issues/8816#issuecomment-1083544957). Maybe let's just be conservative and always `xfail`, regardless of what `fuse` is